README.md

```diff
@@ -49,7 +49,6 @@ fn main2(evl: EventLoop<()>) {
 }
 
 fn main() {
-#   return;
```

Comment:
This is necessary for the tests to work without opening up a window, so I'd like to avoid removing it.

Reply:
Ahh, I see. Sure! Added